common/static/common/js/discussion/discussion_router.js

```diff
@@ -0,0 +1,167 @@
+/* globals DiscussionThreadListView, DiscussionThreadView, DiscussionUtil, NewPostView */
```

Comment:
Ignore this is these lines are in the original JS as well, I can't tell, I'm only reviewing the green diffs.